README.markdown

```diff
@@ -196,3 +196,58 @@ JRE_FILE_PATH=${PROJECT_DIR}/target/framework-package/jdk.tar.gz
 CASSANDRA_FILE_PATH=${PROJECT_DIR}/target/framework-package/cassandra.tar.gz
 
 ```
+
+## Using Cassandra tools
+
+You can use standard command line tools delivered with Apache Cassandra against clusters running on
+Apache Mesos.
+
+This framework provides API endpoints for most tools. All you need is `curl` and the hostname/IP of
+the node running the scheduler of this framework.
+
+An example how to invoke `nodetool`:
+
+```
+cassandra/2.1/bin/nodetool `curl -s http://192.168.5.101:18080/live-nodes/nodetool` status
+Datacenter: datacenter1
+=======================
+Status=Up/Down
+|/ State=Normal/Leaving/Joining/Moving
+--  Address    Load       Tokens  Owns (effective)  Host ID                               Rack
+UN  127.0.0.1  41,12 KB   256     100,0%            dc04253f-d878-4d44-acf3-dc014e058b03  rack1
+UN  127.0.0.2  55,51 KB   256     100,0%            685e82e9-fa26-4d12-bee3-13d4f823f5c9  rack1
+```
+
+More examples (note that you have to adjust the path `cassandra/2.1` to where your Apache Cassandra
+distribution lives):
+
+```
+cassandra/2.1/bin/cqlsh `curl -s http://192.168.5.101:18080/live-nodes/cqlsh`
+cassandra/2.1/bin/nodetool `curl -s http://192.168.5.101:18080/live-nodes/nodetool` status
+cassandra/2.1/tools/bin/cassandra-stress read `curl -s http://192.168.5.101:18080/live-nodes/stress`
+```
+
+There are also two endpoints - one returns a simple JSON structure and one just plain ASCII with the native port
+in the first line and node IP addresses on each following line. The number as the last part of the path determines
+the number of nodes you'd like to have.
+
+Example for `json` endpoint:
+```
+curl -s http://192.168.5.101:18080/live-nodes/json/2
+{
+  "nativePort" : 9042,
+  "rpcPort" : 9160,
+  "jmxPort" : 7199,
+  "liveNodes" : [ "127.0.0.2", "127.0.0.1" ]
+}
+```
+
+Example for `ascii` endpoint:
```

Comment:
I'm having a hard time imagining a case where ascii would be useful. If you don't have a use case in mind please remove it.